SU2_CFD/src/drivers/CDriver.cpp

```diff
@@ -3737,7 +3738,7 @@ void CDriver::StartSolver(){
 
     /*--- Perform a dynamic mesh update if required. ---*/
 
-      if (!fem_solver) {
+      if (!fem_solver && !(config_container[ZONE_0]->GetGrid_Movement() && config_container[ZONE_0]->GetDiscrete_Adjoint())) {
```

Comment:
The StartSolver() in CDriver should be deprecated soon. Has this feature also been tested with the single-zone driver, using `SINGLEZONE_DRIVER = YES`? This should become the default once #724 is merged.

Reply:
I've now tested this with SINGLEZONE_DRIVER = YES and NO, the primal result is identical. The added test case has the setting SINGLEZONE_DRIVER= YES.